R/qc_ids.R

```diff
@@ -35,14 +35,14 @@
 #' )
 #' }
 qc_ids <-
-  function(data, sample, grouping, intensity, remove_na_intensities = TRUE, condition = NULL, title = "ID count per sample", plot = TRUE, interactive = TRUE) {
+  function(data, sample, grouping, intensity = NULL, remove_na_intensities = TRUE, condition = NULL, title = "ID count per sample", plot = TRUE, interactive = TRUE) {
     protti_colors <- "placeholder" # assign a placeholder to prevent a missing global variable warning
     utils::data("protti_colors", envir=environment()) # then overwrite it with real data
     if(remove_na_intensities == TRUE){
```

Comment:
This should theoretically resolve the issue. See comment below.
```suggestion
    if(remove_na_intensities == TRUE){
    if(missing(intensity)) stop("Please provide a column containing intensities or set remove_na_intensities to FALSE")
    data <- data %>%
    tidyr::drop_na({{intensity}})
    }
```